app/code/Magento/Customer/Block/Adminhtml/Edit/BackButton.php

```diff
@@ -32,6 +52,12 @@ public function getButtonData()
      */
     public function getBackUrl()
     {
+        if ($fromPath=$this->decoder->decode($this->form->getRequest()->getParam('fromPath'))) {
+            return $this->getUrl(
+                $fromPath,
+                ['id' => $this->form->getRequest()->getParam('review_id')]
```

Comment:
I believe we need a universal solution for similar cases instead of fixing particular places. We see the same situation when we go to the customer information page from the order view page. So, we may have a couple more places in the admin panel where we have the same issue with the back button. 

It would be great if we could find out how to solve this issue for all occurrences. 

Hope it makes sense

Reply:
Hello, @rogyar! I've idone some investigation and I don't see the possibility to make a universal and reasonable  solution to all "Back" buttons.

Reply:
Hi @engcom-Foxtrot. I've been reflected on that for some time and you are right. It's not an easy task unless we have the same name for an entity identifier. 

Anyway, thank you for checking!